### PR TITLE
Refactor/cobra06

### PR DESCRIPTION
**Pull request by hredestig**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,17 +24,14 @@
...
-        'cobra>=0.4',
-        'ipython>=4.0',
-        'pyzmq>=14.5',
-        'numpy>=1.9.2',
-        'bokeh>=0.11.0',
-        'lxml>=3.4.4',
-        'six>=1.9.0',
-        'ipywidgets>=4.0',
-        'pandas>=0.16.2'
+        'cameo',
```
Comment (joaocardoso) [inline setup.py]: cameo is no longer a requirement

File: driven/flux_analysis/fit_objective/optimization.py [hunk 2]
```diff
@@ -61,8 +61,8 @@ def __init__(self, profiles=[], evaluators=[], binary=True, use_reactions=True,
 
         self.kwargs = dict(use_reactions=use_reactions)
         self.kwargs.update(kwargs)
-        self.profiles = profiles
-        self.evaluators = evaluators
+        self.profiles = profiles or []
```
Comment (joaocardoso) [inline driven/flux_analysis/fit_objective/optimization.py]: nicely done 👍

File: setup.py (hunk 1 above)
Comment (hredestig) [inline setup.py]: No we can't because still using time-machine on ExpressionProfile which doesn't have a context manager..

**Comment by hredestig** [comment]
Alright @joaocardoso cameo 0.11.0 has been released so this can be merged now if you like. Made small change, let's see what CI says. Might be that it fails because 0.11.0 did reach pypi yet

**Pull request merged**